tools/taskcluster/get-codecov-token.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This appears to be patterned after the other `get-*-token.py` files in this directory: if so, it'd help the reviewer if you mentioned that in the extended commit summary or in the comments directly in this file so that they can pattern match rather than checking every line diligently.